### PR TITLE
lisa-894-api-spec: Added lifeEventId for 409 error scenarios on all l…

### PR DESCRIPTION
**Pull request by shravannallagoni**
…ifevent api's

### CODE REVIEW

File: resources/public/api/conf/2.0/testdata/fund-release-extension.md [hunk 1]
```diff
@@ -533,6 +534,7 @@
 {
   "code": "FUND_RELEASE_SUPERSEDED",
   "message": "This fund release has already been superseded"
+  "lifeEventId": "6789000001"
```
Comment (chriswaugh) [inline resources/public/api/conf/2.0/testdata/fund-release-extension.md]: Please can you check and confirm this with Trevor? I would expect the lifeEventId returned for FUND_RELEASE_SUPERSEDED would be the ID for the updated fund release, not the ID for the extension?

File: resources/public/api/conf/2.0/testdata/fund-release-outcome.md [hunk 2]
```diff
@@ -279,7 +281,8 @@
         <pre class="code--block">
 {
   "code": "FUND_RELEASE_SUPERSEDED",
-  "message": "This fund release has already been superseded"
+  "message": "This fund release has already been superseded",
+  "lifeEventId": "5678900001"
```
Comment (chriswaugh) [inline resources/public/api/conf/2.0/testdata/fund-release-outcome.md]: As with previous comment, I think this should be the ID for the fund release, please can you confirm with Trevor?

File: resources/public/api/conf/2.0/testdata/fund-release-extension.md (hunk 1 above)
Comment (shravannallagoni) [reply resources/public/api/conf/2.0/testdata/fund-release-extension.md]: I have made the changes, can you please review. Thanks :)

File: resources/public/api/conf/2.0/testdata/fund-release-outcome.md (hunk 2 above)
Comment (shravannallagoni) [reply resources/public/api/conf/2.0/testdata/fund-release-outcome.md]: I have made the changes, can you please review. Thanks :)

File: resources/public/api/conf/2.0/testdata/fund-release-extension.md [hunk 3]
```diff
@@ -533,6 +534,7 @@
 {
   "code": "FUND_RELEASE_SUPERSEDED",
   "message": "This fund release has already been superseded"
+  "lifeEventId": "3456789001"
```
Comment (chriswaugh) [inline resources/public/api/conf/2.0/testdata/fund-release-extension.md]: This is perfect, but in reviewing this I now release the fund release id in the request body needs to be 3456789000. Sorry!

File: resources/public/api/conf/2.0/testdata/fund-release-outcome.md [hunk 4]
```diff
@@ -279,7 +281,8 @@
         <pre class="code--block">
 {
   "code": "FUND_RELEASE_SUPERSEDED",
-  "message": "This fund release has already been superseded"
+  "message": "This fund release has already been superseded",
+  "lifeEventId": "3456789001"
```
Comment (chriswaugh) [inline resources/public/api/conf/2.0/testdata/fund-release-outcome.md]: As with extensions - this change is correct, but the fund release id in the request needs updating to be 3456789000

**Comment by chriswaugh** [comment]
@shravannallagoni also the extension 1 already approved and extension 2 already approved errors in the create an extension API also return life event ids, so we need to cater for these too.

**Comment by chriswaugh** [comment]
All LGTM :+1:

**Pull request merged**